http_check/datadog_checks/http_check/http_check.py

```diff
@@ -248,7 +254,9 @@ def report_as_service_check(self, sc_name, status, instance, msg=None):
         # Only add the URL tag if it's not already present
         if not any(filter(re.compile('^url:').match, tags)):
             tags.append('url:{}'.format(url))
+        return tags
 
+    def report_as_service_check(self, sc_name, status, tags, msg=None):
         if sc_name == self.SC_STATUS:
```

Comment:
Always true as well.

Reply:
In line 240 we have: `service_checks.append((self.SC_SSL_CERT, status, msg))`